src/areaSearch/areaSearchApplicationPage.tsx

```diff
@@ -27,127 +26,135 @@ interface State {
 
 interface Props extends State {
   openLoginModal: () => void;
+  setNextStep: any;
 }
 
 const AreaSearchApplicationPage = ({
   openLoginModal,
   isSubmittingAreaSearch,
   lastSubmission,
   isFormValid,
+  setNextStep,
 }: Props): JSX.Element => {
   const { t } = useTranslation();
-  const navigate = useNavigate();
   const [isSaveClicked, setSaveClicked] = useState<boolean>(false);
 
   const openPreview = () => {
     setSaveClicked(true);
 
     if (isFormValid) {
-      navigate(getRouteById(AppRoutes.AREA_SEARCH_APPLICATION_FORM_PREVIEW));
+      setNextStep();
     }
   };
 
   return (
-    <AuthDependentContent>
-      {(loading, loggedIn) => {
-        return (
-          <>
-            <MainContentElement className="ApplicationPage">
-              <Helmet>
-                <title>
-                  {getPageTitle(
-                    t(
-                      'areaSearch.application.form.pageTitle',
+    <>
+      <ScrollToTop />
+      <AuthDependentContent>
+        {(loading, loggedIn) => {
+          return (
+            <>
+              <MainContentElement className="ApplicationPage">
+                <Helmet>
+                  <title>
+                    {getPageTitle(
+                      t(
+                        'areaSearch.application.form.pageTitle',
+                        'Area search application',
+                      ),
+                    )}
+                  </title>
+                </Helmet>
+                <Container>
+                  <h1>
+                    {t(
+                      'areaSearch.application.heading',
                       'Area search application',
-                    ),
-                  )}
-                </title>
-              </Helmet>
-              <Container>
-                <h1>
-                  {t(
-                    'areaSearch.application.heading',
-                    'Area search application',
-                  )}
-                </h1>
-                {lastSubmission && <AreaSearchTargetSummary />}
-                <div className="ApplicationPage__form-container">
-                  {loading || isSubmittingAreaSearch ? (
-                    <BlockLoader />
-                  ) : (
-                    <>
-                      {loggedIn ? (
-                        <>
-                          {!lastSubmission ? (
+                    )}
+                  </h1>
+                  {lastSubmission && <AreaSearchTargetSummary />}
+                  <div className="ApplicationPage__form-container">
+                    {loading || isSubmittingAreaSearch ? (
+                      <BlockLoader />
+                    ) : (
+                      <>
+                        {loggedIn ? (
+                          <>
+                            {!lastSubmission ? (
```

Comment:
This nesting is a bit unreadable perhaps, any way to clear it up, like putting it in a function in which you can use if else?